tests/test_fips.py

```diff
@@ -72,6 +72,14 @@ def test_enable_fips_installed_not_enabled(self):
             'Please reboot into the FIPS kernel to enable it.',
             process.stderr.strip())
 
+    def test_enable_fips_not_all_packages_installed(self):
+        # one of the packages is not installed
+        self.make_fake_binary(
+            'dpkg-query', command='[ $2 != openssh-client-hmac ]')
+        process = self.script('enable-fips', 'user:pass')
+        self.assertEqual(process.returncode, 0)
+        self.assertIn('Successfully configured FIPS', process.stdout)
```

Comment:
I suggest to also check for `Installing FIPS packages`.

Reply:
Done